ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java

```diff
@@ -12,33 +12,33 @@
 
 import java.util.NoSuchElementException;
 
+import static be.vlaanderen.informatievlaanderen.ldes.ldio.config.LdioMetricValues.*;
 import static org.springframework.web.reactive.function.server.RequestPredicates.POST;
 import static org.springframework.web.reactive.function.server.RouterFunctions.route;
 
-public class LdioHttpIn extends LdiInput {
+public class LdioHttpIn extends LdioInput {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(LdioHttpIn.class);
 	private final String endpoint;
 
-	public LdioHttpIn(ComponentExecutor executor, LdiAdapter adapter, String endpoint) {
-		super(executor, adapter);
-		this.endpoint = endpoint;
+	public LdioHttpIn(String componentName, String pipelineName, ComponentExecutor executor, LdiAdapter adapter) {
+		super(componentName, pipelineName, executor, adapter);
+		this.endpoint = pipelineName;
 	}
 
 	public RouterFunction<ServerResponse> mapping() {
 		return route(POST("/%s".formatted(endpoint)),
 				req -> {
+					Metrics.counter(LDIO_DATA_IN, PIPELINE, pipelineName, LDIO_COMPONENT_NAME, componentName).increment();
```

Comment:
Won't this result in double counts? We call processInput below, which calls processModel, where we have the same counter line..